guids-generator.html

```diff
@@ -456,17 +497,50 @@
                         />
                     </Grid>
                     <Grid item>
-                        <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
+                      <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
                     </Grid>
                 </Grid>
                 { Object.keys(bhashes).length > 0 &&
-                    <List>
-                        { validatedData.map( item =>
-                            <ListItem className={classes.listItem}>
-                                <ListItemText primary={bhashes[item.index].value} style={{color: bhashes[item.index].isError ? 'red' : 'black'}} />
-                            </ListItem>
-                        )}
-                    </List>
+                    <TableContainer component={Paper}>
+                        <Table className={classes.table} size="small">
+                            <TableHead>
+                              <TableRow>
+                                <StyledTableCell>Health card #</StyledTableCell>
+                                <StyledTableCell>Province</StyledTableCell>
+                                <StyledTableCell>Date of birth</StyledTableCell>
+                                <StyledTableCell>
+                                  <Grid container direction="row" justify="space-between" alignItems="center">
+                                    <Grid item>GUID</Grid>
+                                    <Grid item>
+                                      <CopyToClipboard text={validatedData.map(item => item.isError ? '' : bhashes[item.index].value).join('\r\n')}
+                                                   onCopy={() => onCopyButtonClick()}
+                                                   options={{"format" : "text/plain", "debug" : true}}
```

Comment:
Do we still need debug: true?
```suggestion
                                                   options={{"format" : "text/plain"}}
```

Reply:
Imho when using external libraries always good to have debug just in case, does not hurt & user does not see it. Same as console.err() on hash errors.